### ISSUE TITLE
justified usecase: for access with files in a folder starting with a dot

### ISSUE THREAD

**Issue by davidnuescheler** [adobe/helix-pages#240]
> I think that the 403 from static is accurate, if inconvenient and scary to the visitor. But come on, what should a visitor expect that is trying to sniff around in our `.env` files?
> _Originally posted by @trieloff in https://github.com/adobe/helix-pages/issues/218#issuecomment-605012892_

i just realized that apple pay requires access to a `.well-known` folder for domain verification.

https://developer.apple.com/documentation/apple_pay_on_the_web/maintaining_your_environment#3179140

i am not sure what would be the best way forward on this, any ideas? this may or may not be the only place where this happens, so i wonder if we should revisit the blacklisting of all `.` folder and be more selective or if we should white-list things selectively over the `.` blacklist.

**Comment by trieloff** [comment]
.well_known should get special treatment.

**Comment by davidnuescheler** [comment]
this one still doesn't seem to work, i don't know exactly why, possibly because the apple pay verification file doesn't have an extension...

**Comment by trieloff** [comment]
Yes, that's the likely reason. We can fix it in `helix-dispatch` with another request.
